sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultPodSpec.java

```diff
@@ -455,6 +464,23 @@ public Builder sharePidNamespace(Boolean sharePidNamespace) {
             return this;
         }
 
+        /**
+         * Sets the {@code host volumes} and returns a reference to this Builder so that the methods can be
+         * chained together.
+         *
+         * @param hostVolumes the {@code hostVolumes} to set
+         * @return a reference to this Builder
+         */
+        public Builder hostVolumes(Collection<HostVolumeSpec> hostVolumes) {
+            if (hostVolumes == null) {
```

Comment:
FWIW I'm not sure why this builder has special behavior for passing null values, but given the rest of the file does this I guess it doesn't hurt to do the same here for now.